src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -218,12 +240,13 @@ export class MeetingManager implements AudioVideoObserver {
   }
 
   createLogger(configuration: MeetingSessionConfiguration) {
-    const consoleLogger = new ConsoleLogger('SDK', this.logLevel);
-    let logger: ConsoleLogger | MultiLogger = consoleLogger;
+    let logger: Logger = this.logger
```

Comment:
I think if they pass their own logger, we should not call createLogger at all

Reply:
Yeah, I think it makes sens, cause user is able to pass a`MultiLogger`. No need to call `createLogger`.